Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7963.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals; 
+
+namespace Xamarin.Forms.Controls.Issues
+{ 
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 7963, "No shadows for UWP Frames (and WPF)?", PlatformAffected.WPF | PlatformAffected.UWP)]
```

Comment:
This is only WPF, right?

Reply:
There is issue #7839 which is similiar for UWP

Reply:
But, this one is only for WPF... Right? 😉 

I don't really care about the title, but the affected platforms now has UWP while that is not fixed in this PR. Whenever a PR comes up for UWP it can be added here or there will be a new one.

Although these flags do not necessarily have much impact at this time, I just like to keep them in check for when they do get meaning.

Reply:
Done!